infra/capi/cluster-capi-infrastructure-ironic/cluster-api-infrastructure-ironic.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: v1
+data:
+  CACHEURL: http://172.22.0.1/images
+  DEPLOY_KERNEL_URL: http://172.22.0.2:6180/images/ironic-python-agent.kernel
+  DEPLOY_RAMDISK_URL: http://172.22.0.2:6180/images/ironic-python-agent.initramfs
+  DHCP_HOSTS: b4:96:91:c0:31:64,id:*;b4:96:91:c4:e3:f4,id:*
+  DHCP_IGNORE: tag:!known
+  DHCP_RANGE: 172.22.0.10,172.22.0.100
+  DNS_IP: 172.22.0.3
+  GATEWAY_IP: 172.22.0.4
+  HTTP_PORT: "6180"
+  IRONIC_BASE_URL: http://172.22.0.2:6385
+  IRONIC_ENDPOINT: http://172.22.0.2:6385/v1/
+  IRONIC_INSPECTOR_VLAN_INTERFACES: all
+  IRONIC_KERNEL_PARAMS: console=ttyS0
+  IRONIC_RAMDISK_SSH_KEY: PLACEHOLDER_SSH_KEY
+  PROVISIONING_INTERFACE: eth2
+  USE_IRONIC_INSPECTOR: "false"
+kind: ConfigMap
+metadata:
+  name: baremetal-operator-ironic-bmo-configmap-6cf9t7484b
```

Comment:
Can we avoid the last dynamic sequence of characters (6cf9t7484b) in this configmap name? Just baremetal-operator-ironic-bmo-configmap should be good